scenarios/writing_clamp_function_roc.yaml

```diff
@@ -1,97 +1,88 @@
 first: '1'
 screens:
   1: |
-    ┃
+    ┃🗌
   2: |
     clamp : 🗌
     clamp┃ = 🗌
   3: |
     clamp : 🗌
-    clamp = ┃
+    clamp = ┃🗌
   4: |
     clamp : 🗌 -> 🗌
-    clamp = \┃ ->
+    clamp = \┃🗌 ->
       🗌
   5: |
     clamp : 🗌 -> 🗌
     clamp = \v┃ ->
       🗌
   6: |
-      clamp : 🗌, 🗌 -> 🗌
-      clamp = \v, ┃ ->
-        🗌
+    clamp : 🗌, 🗌 -> 🗌
+    clamp = \v, ┃🗌 ->
+      🗌
   7: |
     clamp : 🗌, 🗌 -> 🗌
     clamp = \v, low┃ ->
       🗌
   8: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
-    clamp = \v, low, ┃ ->
+    clamp = \v, low, ┃🗌 ->
       🗌
   9: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high┃ ->
       🗌
   10: |
     clamp : 🗌, 🗌, 🗌, 🗌 -> 🗌
-    clamp = \v, low, high, ┃ ->
+    clamp = \v, low, high, ┃🗌 ->
       🗌
   11: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high ->
-      ┃
+      ┃🗌
   12: |
-    clamp : a, *, * -> a
+    clamp : a, 🗌, 🗌 -> a
     clamp = \v, low, high ->
       v┃
   13: |
-    clamp : a, *, * -> a
-    clamp = \v, low, high ->
-      v ┃
-  14: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high ->
       v
-        |> ┃
-  15: |
+        |> ┃🗌
+  14: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high ->
       v
         |> max┃
-  16: |
-    clamp : 🗌, 🗌, 🗌 -> 🗌
+  15: |
+    clamp : Num a, 🗌, 🗌 -> Num a
     clamp = \v, low, high ->
       v
-        |> max ┃
-  17: |
-    clamp : Num a, Num a, * -> Num a
+        |> max ┃🗌
+  16: |
+    clamp : Num a, Num a, 🗌 -> Num a
     clamp = \v, low, high ->
       v
         |> max low┃
-  18: |
-    clamp : Num a, Num a, * -> Num a
```

Comment:
Same thing here, even if it doesn't make sense semantically, its legal syntactically. I think it should be allowed.